pkg/test/tpm.go

```diff
@@ -280,6 +280,39 @@ func PSIndexConfig(txtAPI api.ApiInterfaces) (bool, error, error) {
 		if err != nil {
 			return false, nil, err
 		}
+		err = binary.Read(buf, binary.BigEndian, &d2.NameAlg)
+		if err != nil {
+			return false, nil, err
+		}
+		err = binary.Read(buf, binary.BigEndian, &d2.Attributes)
+		if err != nil {
+			return false, nil, err
+		}
+		// Helper variable hashSize- go-tpm2 does not implement proper structure
+		var hashSize uint16
+		err = binary.Read(buf, binary.BigEndian, &hashSize)
+		if err != nil {
+			return false, nil, err
+		}
+		// Uses hashSize to make the right sized slice to read the hash
+		hashData := make([]byte, hashSize)
+		err = binary.Read(buf, binary.BigEndian, &hashData)
+		if err != nil {
+			return false, nil, err
+		}
+		err = binary.Read(buf, binary.BigEndian, &d2.DataSize)
+		if err != nil {
+			return false, nil, err
+		}
+
```

Comment:
add comment which spec was used (chapter number and name)

Reply:
done